CONTRIBUTING.md

```diff
@@ -150,6 +150,97 @@ cd test/manual/k8s_crds
 ```
 This test currently does not run as part of the test suite.
 
+## Profiling
```

Comment:
A link to this section needs to be added to the table of contents at the top of this file.